app/constants/file-uploader-size-limits.js

```diff
@@ -0,0 +1,6 @@
+export const PLANNING_UNIT_UPLOADER_MAX_SIZE = 1e6; // 1MB
+export const PLANNING_AREA_UPLOADER_MAX_SIZE = 1e6; // 1MB
+export const PLANNING_AREA_GRID_UPLOADER_MAX_SIZE = 1e7; // 10MB
+export const PROTECTED_AREA_UPLOADER_MAX_SIZE = 1e7; // 10MB
+export const FEATURES_UPLOADER_MAX_SIZE = 2e7; // 20MB
+export const COST_SURFACE_UPLOADER_MAX_SIZE = 2e7; // 20MB
```

Comment:
@SARodrigues I suggest to use MiBs rather than MBs here. I know it won't matter in most circumstances, and the spec of defaults in M-1032 wasn't fully clear about the specific measurement unit, but we do actually default to multiple of MiBs in the API implementation (https://github.com/Vizzuality/marxan-cloud/blob/develop/api/apps/api/src/modules/uploads/upload-limits.ts) - and I'd say we should actually limit the configurability of these limits to MiBs in the API too, if sticking to this works for you. It should simplify how we handle these limits, as I definitely don't expect anything sub-MiB (or fractions of MiB) to ever be needed nor make sense.

Reply:
@hotzevzl Updated to use MiBs! I added a note to the respective frontend file with a link to the backend counterpart, to prevent any confusion in the future. :)